schemas/session-replay/mobile/mutation-data-schema.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "mutation-data-schema.json",
+    "title": "MutationData",
+    "type": "object",
+    "description": "Schema of a MutationData.",
+    "allOf": [
+        {
+            "properties": {
+                "source": {
+                    "type": "integer",
+                    "const": "1",
+                    "description": "The source of this type of incremental data.",
+                    "readOnly": true
+                },
+                "adds": {
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "type": "object",
+                        "required": [
+                            "previousId",
```

Comment:
We need `previousId` to accept `null` value, hence it cannot be `required`. This is to enable placing the new `wireframe` at the beginning of the array.

Ref. in RFC: [Adding new wireframes](https://datadoghq.atlassian.net/wiki/spaces/RUMP/pages/2471460887/RFC+-+Data+Format+for+Mobile+Session+Replay)
> wireframe in draw order (or null if the new wireframe should be rendered at the very bottom).